clang/include/clang/ExtractAPI/API.h

```diff
@@ -256,14 +256,14 @@ struct APIRecord {
   APIRecord() = delete;
 
   APIRecord(RecordKind Kind, StringRef USR, StringRef Name,
-            PresumedLoc Location, AvailabilitySet Availabilities,
+            PresumedLoc Location, const AvailabilityInfo &Availability,
```

Comment:
Can you keep this as a value and bring back the `std::move` below, this allows to prevent unnecessary copies.